src/views/PurchaseOrder.vue

```diff
@@ -17,7 +17,7 @@
         </ion-item> 
         <ion-item lines="none">
           <ion-label>{{ $t("Select mapping") }}</ion-label>
-          <ion-select :disabled="!Object.keys(fieldMappings).length || !file" interface="popover" @ionChange="mapFields">
+          <ion-select :value="Object.keys(fieldMappings).length" :disabled="!Object.keys(fieldMappings).length || !file" interface="popover" @ionChange="mapFields">
```

Comment:
Why we are setting it to length?

Reply:
Yes sir it was unnecessary and creating issue. Fixed it.